EXOSIMS/Observatory/TwoStarShades_mission.py

```diff
@@ -0,0 +1,567 @@
+from EXOSIMS.Observatory.SotoStarshade import SotoStarshade
+import numpy as np
+import astropy.units as u
+import astropy.constants as const
+import scipy.optimize as optimize
+
+
+class TwoStarShades_mission(SotoStarshade):
+    def __init__(
+        self,
+        scMass=[6000.0, 6000.0],
+        dryMass=[3400.0, 3400.0],
+        counter_1=0,
```

Comment:
why are counter_? inputs?  Is there ever a case where they should be initialized as non-zero?

Reply:
initially they're always zero.